src/gui/Layout.cpp

```diff
@@ -50,12 +50,7 @@ void Layout::checkScroll(GtkAdjustment* adjustment, double& lastScroll)
 {
 	XOJ_CHECK_TYPE(Layout);
 
-	if (!view->shouldIgnoreTouchEvents())
```

Comment:
This does issues, therefore I removed it, I plan to add something like this, but I need a better solution.